tools/tpm2_quote.c

```diff
@@ -49,12 +49,12 @@ typedef struct {
 } PCR_LIST;
 
 static TPMS_AUTH_COMMAND sessionData;
-static char *outFilePath;
+static char *outFilePath, *outSigFilePath;
 static TPM2B_DATA qualifyingData = TPM2B_EMPTY_INIT;
 static TPML_PCR_SELECTION  pcrSelections;
 static bool is_auth_session;
 static TPMI_SH_AUTH_SESSION auth_session_handle;
-static int k_flag, c_flag, l_flag, g_flag, L_flag, o_flag;
+static int k_flag, c_flag, l_flag, g_flag, L_flag, o_flag, s_flag;
```

Comment:
You don't really need the s_flag, just check for outSigFilePath (but rename it to not use camel case convention).